Mobile/lib/Components/ColorScheme.dart

```diff
@@ -4,15 +4,22 @@ import 'dart:math';
 class CustomColorScheme {
   static final random = Random();
   
-  static List<Color> colorScheme = const [
-    Color.fromRGBO(194, 232, 255, 1),
-    Color.fromRGBO(137, 213, 107, 1),
-    Color.fromRGBO(240, 110, 81, 1),
+  static Color primary = const Color.fromRGBO(194, 232, 255, 1);
+  static Color secondary = const Color.fromRGBO(245, 197, 58, 1);
+  static Color error = const Color.fromRGBO(238, 100, 118, 1);
+  static Color randomColor = const Color.fromRGBO(137, 213, 107, 1);
+  static Color randomColorTwo = const Color.fromRGBO(240, 110, 81, 1);
+  static Color menu = const Color.fromRGBO(217, 217, 217, 1);
+
+  static List<Color> taskCardColorSheme = [
+    primary,
+    randomColor,
+    randomColorTwo,
```

Comment:
If you update the color naming, then these should also be updated
```suggestion
    limeGreen,
    tangerineOrange,
```